sdk-libs/token-sdk/README.md

```
@@ -0,0 +1,74 @@
+<!-- cargo-rdme start -->
+
+## Light Token SDK
+
+The base library to use Light Token Accounts, Light Mints, and compressed token accounts.
+
+### Light Token Accounts
```

markdownlint-cli2 (0.20.0)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)